### DISCUSSION TITLE
Detect save extension by name

### DISCUSSION THREAD

**Discussion by xdanielc** [tannerhelland/PhotoDemon#500] [Ideas]
Just like the title says, this is something photoshop doesn't have but gimp does.
You just tipe the name extension when saving and that overrides the dropdown for the next popup settings for the output format.

Just wanna say I love this program, is the fastest alternative I've found for when you wanna do a small retouch but loading those two is too much.

**Comment by tannerhelland** [comment]
Hi @xdanielc .  This feature is available in current PhotoDemon nightly builds.  Give it a try and let me know how it works for you!

(I am currently wrapping up work on [JPEG XL](https://en.wikipedia.org/wiki/JPEG_XL) support, and once that is done I plan to release a new stable build that includes the new "detect save extension by name" feature.)
